backend/fms_core/utils.py

```diff
@@ -88,6 +88,13 @@ def get_normalized_str(d: dict, key: str, default: str = "") -> str:
     """
     return str_cast_and_normalize(d.get(key) or default)
 
+
+def is_date_after_today(date):
+    if not isinstance(date, datetime.date):
+      return None
```

Comment:
I wonder if we should raise an exception here - it might help catch bugs.

Reply:
it's an utility i don't want to propagate errors each time i use this. If it breaks we'll fix it.